debian/control

```diff
@@ -39,3 +39,19 @@ Package: kano-updater-i18n-orig
 Architecture: all
 Description: Data for working on translations of kano-updater
 Multi-Arch: foreign
+
+Package: kano-os-pip-transition
```

Comment:
Not too keen on this package name - I know that right now we are doing it to transition from pip to Debian packages but really it is just a metapackage of pip modules which are required to be installed. Do we even want this package - should these dependencies not just go into the `kano-os` metapackage?

Reply:
The idea of the subpackage is to temporarily help transition `python_modules` with a debian way. Ideally, all of the depends of this one would be transitioned to other packages that actually need them.

Reply:
That's it, mostly a temporary package, to safeguard that everything works as planned and is easier to debug. For the next release, it shouldn't be there; moreover attach the dependencies directly on the updater control.

I could add a comment there, if there was a way to add comments in a `debian/control` file 🤔